src/backend/api/Fusion.Resources.Api/Controllers/Requests/AnalyticsController.cs

```diff
@@ -45,6 +47,40 @@ public async Task<ActionResult<ApiCollection<ApiResourceAllocationRequestForAnal
             return collection;
         }
 
+        [MapToApiVersion("2.0")]
+        [HttpGet("/analytics/requests/internal")]
+        public async Task<ActionResult<IAsyncEnumerable<ApiResourceAllocationRequestForAnalytics>>> GetAllRequestsV2([FromQuery] ODataQueryParams query)
+        {
+            #region Authorization
+
+            var authResult = await Request.RequireAuthorizationAsync(r =>
+            {
+                r.AlwaysAccessWhen().FullControl().FullControlInternal();
+                r.AnyOf(or =>
+                {
+                    or.GlobalRoleAccess("Fusion.Analytics.Requests");
+                });
+            });
+
+            if (authResult.Unauthorized)
+                return authResult.CreateForbiddenResponse();
+
+            #endregion
+
+            var requestQuery = DispatchAsync(new GetResourceAllocationRequestsForAnalyticsAsync(query));
```

Comment:
Move await to DispatchAsync?
```suggestion
            var requestQuery = await DispatchAsync(new GetResourceAllocationRequestsForAnalyticsAsync(query));
```